codebase/scss/components/_footer.scss

```diff
@@ -4,53 +4,69 @@ div.info-footer-wrapper {
   padding: 20px;
   margin: 0 auto;
   width: 100%;
+
+  #infoFooterBlock {
+    width: 90%;
+    margin: 0 5%;
+  }
+
   h3 {
     text-transform: uppercase;
-    color: white;
+    color: $white;
```

Comment:
SUGGESTION/TRIVIAL: Unless this is a specific white, I think it's fine to leave it with the default, not with the variable. The only big reason to set it to the variable would be if the entire sites white was going to change. For primary colors, I kinda think it's fine to leave it at default

Reply:
I think you're right here, I'll revert.